stream_alert/rule_processor/parsers.py

```diff
@@ -139,27 +145,67 @@ def _key_check(self, schema, json_records):
                     # Nested key check
                     if key_type and isinstance(key_type, dict):
                         schema_match = self._key_check(schema[key], [json_records[index][key]])
-            else:
-                LOGGER.debug('JSON Key mismatch: %s vs. %s', json_keys, schema_keys)
 
             if not schema_match:
+                LOGGER.debug('Schema: \n%s', json.dumps(schema, indent=4))
+                LOGGER.debug('Key Check Failure: \n%s', json.dumps(json_records[index], indent=4))
```

Comment:
I don't think these logger messages need to be all title case and should just read like sentences:
`Key check failure: ...` and `Missing keys in record: ...` There are a few other logger calls like this throughout too that should be addressed for uniformity